apps/test/unit/lib/kits/maker/boards/microBit/ExternalLedTest.js

```diff
@@ -0,0 +1,103 @@
+import {expect} from '../../../../../../util/reconfiguredChai';
```

Comment:
nit: I think we typically use the `'@cdo/...'` imports for tests to avoid the long filepaths (see import ExternalLed below)

Reply:
Since `@cdo/apps` maps to `apps/src`, I don't think I can use this for `reconfiguredChai` that lives at `apps/test/util`. Is that correct? I looked at other test files that import `reconfiguredChai` and don't see the `@cdo` convention used.

Reply:
Ahh yep you're right, I misread for a second due to how long the path was.